src/applications/static-pages/caregiver-content-toggle/widget.jsx

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
-import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import { connect } from 'react-redux';
 
 import OnState from './On';
-import OffState from './Off';
 
-function CaregiverContentToggle({ showLinkToOnlineForm }) {
-  if (showLinkToOnlineForm) {
-    return <OnState />;
-  }
-
-  return <OffState />;
+function CaregiverContentToggle() {
+  return <OnState />;
```

Comment:
Since this is ultimately going to be completely static content, I think we should let the editors know so that we can just add that content onto the actual page in the CMS, then we can just delete this whole directory

Reply:
@ncksllvn this was on hold for weeks and never got the go-ahead from stakeholders... so I was planning on asking once we got approval, thanks for reaching out!